modules/invibesBidAdapter.js

```diff
@@ -87,10 +109,25 @@ function buildRequest(bidRequests, bidderRequest) {
   const _placementIds = [];
   const _adUnitCodes = [];
   let _customEndpoint, _userId, _domainId;
-  let _ivAuctionStart = bidderRequest.auctionStart || Date.now();
+  let _ivAuctionStart = Date.now();
+  if (window.placementIds == undefined) {
+    window.placementIds = [];
+  }
```

Comment:
Could this be updated to a more specific namespace?

Reply:
Suggest this is updated to remove the conditional check and always set either the existing value or set to an empty array:
`window.placementIds = window.placementIds || [];`

Reply:
Thanks. I will do the change.

Reply:
Could you also update the namespace to be specific of your adapter?

```
window.invibes = window.invibes || {};
window.invibes.placementIds = window.invibes.placementIds || [];
```